packages/chevrotain/src/parse/cst/cst_visitor.ts

```diff
@@ -132,19 +132,23 @@ export function validateMissingCstMethods(
   visitorInstance: ICstVisitor<unknown, unknown>,
   ruleNames: string[]
 ): IVisitorDefinitionError[] {
-  const errors: IVisitorDefinitionError[] = map(ruleNames, (currRuleName) => {
+  const errors = flatMap(ruleNames, (currRuleName) => {
```

Comment:
Would it be more readable if we change it to two separate steps (in two separate statements).
- `filter` to detect only missing methods 
- `map` missing method names to the errors

?

Reply:
Maybe. I think it's fine as is, we have this pattern to compute errors in a lot of places, it shouldn't be too confusing.